assets/stylesheets/main.scss

```diff
@@ -59,3 +61,4 @@
 @import '../pages/oneoff-contributions/oneoffContributions';
 @import '../pages/regular-contributions-existing/regularContributionsExisting';
 @import '../pages/paypal-error/payPalError';
+@import '../pages/error/error';
```

Comment:
Can we avoid adding pages to the `main.scss` and only serve the CSS that we are going to use on that page?